app/assets/javascripts/applicants.js.coffee.erb

```diff
@@ -0,0 +1,45 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://jashkenas.github.com/coffee-script/
+jQuery ->
+  $applicantTable = $('#applicants').dataTable
+                      columnDefs: [
+                                    { orderable: false}
```

Comment:
Does columnDefs need a target ? I know the `columns` style doesn't, it based on position.
https://datatables.net/reference/option/columnDefs